### PR TITLE
Remove dependency on gofrs/uuid

### PR DESCRIPTION
**Pull request by Lyndon-Li**
Remove the unnecessary dependency on gofrs/uuid to reduce security risks

### CODE REVIEW

File: internal/hook/item_hook_handler.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import (
 	"strings"
 	"time"
 
-	uuid "github.com/gofrs/uuid"
+	uuid "github.com/google/uuid"
```
Comment (reasonerjt) [inline internal/hook/item_hook_handler.go:25]: not quite sure why we need to rename this.
why not just make it 
"github.com/google/uuid" 
in the import chunk?

Comment (Lyndon-Li) [reply internal/hook/item_hook_handler.go:25]: They are different library, or two different provider of uuid. I am not sure why `gofrs/uuid` was used either, but as far as my understanding, it is safe to replace it with `google/uuid`

Comment (Lyndon-Li) [reply internal/hook/item_hook_handler.go:25]: Not sure the history, but IMO, we can rename it

Comment (Lyndon-Li) [reply internal/hook/item_hook_handler.go]: Done

**Review by reasonerjt — APPROVED** [review]
lgtm thanks

**Review by blackpiglet — APPROVED** [review]

**Pull request merged**